.github/ISSUE_TEMPLATE/company-showcase-request.md

```diff
@@ -1,41 +0,0 @@
----
```

Comment:
I think you didn't rebase correctly upon master which resulted in the deletion of this file. Can you undo its deletion?

Reply:
I've added it back